README.md

```diff
@@ -9,9 +9,23 @@ fulcio only signs short-lived certificates that are valid for under 20 minutes.
 Fulcio is a *work in progress*.
 There's working code and a running instance and a plan, but you should not attempt to try to actually use it for anything.
 
-The fulcio root cert running on our public instance (https://fulcio.sigstore.dev) is currently:
+The fulcio root certificate running on our public instance (https://fulcio.sigstore.dev) can be obtained and verified against Sigstore's root (at the [sigstore/root-signing](https://github.com/sigstore/root-signing) repository). To do this, install and use [go-tuf](https://github.com/theupdateframework/go-tuf)'s CLI tools:
+```
+$ go get github.com/theupdateframework/go-tuf/cmd/tuf
+$ go get github.com/theupdateframework/go-tuf/cmd/tuf-client
+```
+
+Then, obtain trusted root keys for Sigstore. This can be done from a cosign checkout or directly from Sigstore's root signing repository at a trusted commit.
+```
+$ git clone https://github.com/sigstore/cosign
```

Comment:
It feels cyclical to include a reference to cosign in Fulcio for verification. Can we use the root-signing repository instead?

Reply:
That also felt cyclical since that was the remote repository  :D

OR I could say pin at, let's say, the live stream's publications hash?

Reply:
Yea, that sounds good.

Reply:
done! thanks